src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/02 Advanced Examples.md

```diff
@@ -3,11 +3,22 @@ title: 'Advanced Examples'
 excerpt: 'Advanced Examples using the k6 Scenario API - Using multiple scenarios, different environment variables and tags per scenario.'
 ---
 
-## Using multiple scenarios
+You can use multiple scenarios in one script, and these scenarios can be run in sequence or in parallel.
+So besides helping you model workloads, scenarios can also help you make more flexible and composable test logic.
+Some ways that you can use scenarios include the following:
+- Sequence workloads
+- Add per-scenario tags and environment variables
+- Make scenario-specific thresholds.
+- Run scenarios in different VU [lifecycle functions](/using-k6/test-lifecycle),
 
-This configuration will first execute a scenario where 50 VUs will try to run as many iterations
-as possible for 30 seconds. It will then transition to the next scenario, executing 100 iterations
-per VU for a maximum duration of 1 minute.
+
+## Sequence multiple scenarios
+
+To sequence scenarios, you can use `startTime` property (in combination with the options specific to your scenario executor).
+
+This configuration first executes a scenario where 50 VUs try to run as many iterations
+as possible for 30 seconds.
+It then runs the next scenario, which executes 100 iterations per VU for a maximum duration of 1 minute.
```

Comment:
```suggestion
This configuration first executes a scenario where 50 VUs try to run as many iterations as possible for 30 seconds. It then runs the next scenario, which executes 100 iterations per VU for a maximum duration of 1 minute.
```

Please don't insert extra newlines in the middle of paragraphs. It makes reading the code in different resolutions and making change suggestions much worse, see https://github.com/grafana/k6-docs/pull/229#discussion_r585350973

Reply:
![example](https://user-images.githubusercontent.com/3336773/207317195-5ed95524-8d93-4134-8a5d-32e2da956ed6.png)
e.g. this is how a diff in one of the lines below can look like, if I tile my windows ~60:40

Reply:
Can't believe I somehow deleted my entire screed about this holy war. In short, I disagree. For text I try to break lines on the sentence level, with splits on other punctuation if it gets too long.

First, for reasons of writer comfort. [Asciidoctor has a good list](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line). To expand a bit:
- In modal editors, it's much easier to yank sentences in, out, up and down if they are own each line.
- It provides visual cue about bloated paragraphs. More than 5 sentences is getting dense for our docs. It's easy to track, just count 5 lines.
- It provides visual cues about bloated sentences. 80 chars, the standard terminal space, makes up about 14 words in an English sentence. By coincidence, this is a good average to shoot for simple sentences (complex sentences, I.e. those that have dependant clauses, e.g."If you want to..., then you can...", can be longer).
- It avoids awful diffs from editor reflows.

About diffs, one sentence per line makes diffs much easier to look at, because the changes appear more closely together. Compare a commit from this PR with a commit I made on March 17. Each changes only a punctuation mark:
![image](https://user-images.githubusercontent.com/47385188/207360327-3cfd3171-5db4-4689-b942-1802464d05e3.png)


Sorry, I don't understand what you mean 60:40? Is that line length? I think 80 is a reasonable expectation. I do see that it's annoying to soft wrap. But I favor going farther the other way, with [semantic line breaks](https://sembr.org/). That way, the editors could look at every constituent of the sentence on the molecular level. However, I haven't reached this level of enlightment yet.

Reply:
One sentence per line is mostly ok, but the screenshot I posted had a bunch of breaks mid-sentence. 

I still think the visual cues you'd get if your raw text was similar to how it will appear when the markdown is rendered are more relevant than any you'd get if you split the sentences apart, but it's not that big of a deal :man_shrugging: